TESTS/mbed_drivers/ticker/main.cpp

```diff
@@ -87,61 +89,86 @@ void wait_and_print() {
 }
 
 void test_case_1x_ticker() {
-    led1 = 0;
-    led2 = 0;
-    ticker_count = 0;
-    ticker1->attach_us(ticker_callback_0, ONE_SECOND_MS);
-    wait_and_print();
-}
 
-void test_case_2x_ticker() {
-    led1 = 0;
-    led2 = 0;
-    ticker_count = 0;
-    ticker1->attach(&ticker_callback_1, 1.0);
-    ticker2->attach(&ticker_callback_2_led, 2.0);
-    wait_and_print();
+    char _key[11] = { };
+    char _value[128] = { };
+    uint8_t results_size = 0;
+
+    greentea_send_kv("timing_drift_check_start", 0);
+    ticker1->attach_us(&ticker_callback_0, ONE_MILLI_SEC);
+
+    // wait for 1st signal from host
+    greentea_parse_kv(_key, _value, sizeof(_key), sizeof(_value));
```

Comment:
Please call this in a loop, double checking that the `key` is what you expect it to be.